dev/release/publish-to-maven.sh

```diff
@@ -80,7 +80,7 @@ then
 fi
 
 # Default GPG command to use
-GPG="gpg"
+GPG="gpg --pinentry-mode loopback"
```

Comment:
I got gpg timeouts before I added this

Reply:
Thanks for fixing this @andygrove, I should have done this in the first place.